projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -417,9 +422,33 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 	if ( ( new Host() )->is_wpcom_simple() ) {
 		\ExPlat\assign_current_user( $aa_test_name );
 		$is_enabled = 'treatment' === \ExPlat\assign_current_user( $experiment_name );
+
+		if ( is_automattician() ) {
+			$is_enabled = true;
+			update_user_option( get_current_user_id(), RDV_EXPERIMENT_FORCE_ASSIGN_OPTION, 'treatment', true );
```

Comment:
Just note that the limitationhere is that it will only work with A/B experiments. If you're experiment will have different naming for `treatment` this will fail.